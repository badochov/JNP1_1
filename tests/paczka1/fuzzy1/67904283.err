Error in line 1: E2tEJ78mb A659 24,5
Error in line 5: fczwUavzH S213 2,2
Error in line 13: BVunJdri A402 86,2
Error in line 16: koGf S213 54,3
Error in line 24: UKzKcTJB6C S823 53,5
