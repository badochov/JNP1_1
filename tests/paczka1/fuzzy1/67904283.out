A402 208,0
A402 208,0
7V8atdE A 41,4
BVunJdri A 37,2 S 12,3
E2tEJ78mb A 208,0
fczwUavzH A 58,6
koGf A 18,5
S213 12,3
A402 303,8
A659 59,9
A402 S 121,4
A402 354,2
